graveler/graveler.go

```diff
@@ -218,16 +213,21 @@ type Catalog interface {
 	Merge(ctx context.Context, repositoryID RepositoryID, from Ref, to BranchID) (CommitID, error)
 
 	// DiffUncommitted returns the changes as 'Diff' slice on a repository / branch
-	//   List the differences 'from' path, with 'amount' of result.
-	//   Returns differences found, true (boolean) in case there are more differences - use 'from' with last path from previous call to get the next differences
-	DiffUncommitted(ctx context.Context, repositoryID RepositoryID, branchID BranchID, from Path, amount int) ([]Diff, bool, error)
+	//   List the differences 'from' key, with 'amount' of result.
+	//   Returns differences found, true (boolean) in case there are more differences - use 'from' with last key from previous call to get the next differences
```

Comment:
bool was removed

Reply:
do we still want the _from_ in all methods returning iterators? because one can use SeekGE

Reply:
thought about it - when you implement your iterator did you perform a query on NewIterator or waited for the first Next?

Reply:
waited for next

Reply:
cool - I'll update all